packages/theme-light-extension/style/variables.css

```diff
@@ -407,4 +407,15 @@ all of MD as it is not optimized for dense, information rich UIs.
   --jp-reject-color-normal: var(--md-grey-600);
   --jp-reject-color-hover: var(--md-grey-700);
   --jp-reject-color-active: var(--md-grey-800);
+
+  /* File or activity icons semantic variables */
```

Comment:
We also need to make it work with dark theme, as currently we get a regression:

![Screenshot from 2022-10-28 21-22-54](https://user-images.githubusercontent.com/5832902/198725704-b9efe6a6-81f3-49a2-a58d-f9cbc0911d73.png)

At a minimum we need to:
- add the same set of variables defined for the built-in dark theme
- add an entry in the changelog/extension migration guide documenting that these variables must be defined for themes for JupyterLab 4.0

Reply:
Thnask for the comment. An equivalent set of variables has been added in [2528ff6](https://github.com/jupyterlab/jupyterlab/pull/13279/commits/2528ff6350beb733f0239e42896b6af5acf6b1b5)